flake.nix

```diff
@@ -25,12 +25,12 @@
     # Light-weight wrapper around cardano-node
     ogmios.url = "github:mlabs-haskell/ogmios-nix/v6.11.2";
 
-    cardano-nix.url = "github:mlabs-haskell/cardano.nix";
-
+    # Local Cardano devnet for integration testing
     cardano-devnet-flake.url = "github:szg251/cardano-devnet-flake?ref=szg251/node-10";
```

Comment:
Not directly related to your PR but while I was at it I took a look at the inputs. I see this repo was migrated to `mlabs-haskell`, do you want to also rename it here?

Reply:
nice catch, thank you!